src/client/server_packets.py

```diff
@@ -0,0 +1,23 @@
+from PyQt4 import QtCore
+
+class ServerPackets(QtCore.QObject):
```

Comment:
I'd prefer if we gave this class a bit more responsibility than simply being a dictionary with an aliased `.get(x, None)` function. (This is what `translate` essentially does).

I'd suggest calling it CommandDispatcher and to move client.dispatch in here as well.